### DISCUSSION TITLE
How to make a post?

### DISCUSSION THREAD

**Discussion by yongjian16** [alshedivat/al-folio#1739] [Q&A]
I created a file under '_post' named "2023-9-10-test.md", and the content of it is similar to '2023-07-12-post-bibliography.md'. However, it does not show in the blog. So basically, my question is how to make a post?

**Comment by george-gca** [comment]
Make sure you keep the standard, so the numbers should have 2 digits, with leading zeros when needed. So, the name should be `2023-09-10-test.md`. Also, make sure the same date is present in the post front matter.
